bin/circos_plot.py

```diff
@@ -0,0 +1,204 @@
+            mobilome_track.axis(fc="none", ec="lightseagreen", ls="dashdot", lw=0.15)
+
+        for feature in seqid2features[sector.name]:
+            if feature.type == "CDS":
+                if feature.strand == 1:
+                    f_cds_track.genomic_features([feature], fc="hotpink")
+                else:
+                    r_cds_track.genomic_features([feature], fc="steelblue")
+                if "antismash_bgc_function" in feature.qualifiers:
+                    bgc_track_antismash.genomic_features([feature], fc="tomato")
+                if "gecco_bgc_type" in feature.qualifiers:
+                    bgc_track_gecco.genomic_features([feature], fc="lightsalmon")
+                if "nearest_MiBIG" in feature.qualifiers:
+                    bgc_track_sanntis.genomic_features([feature], fc="firebrick")
+                if "dbcan_prot_type" in feature.qualifiers:
+                    dbcan_track.genomic_features([feature], fc="forestgreen")
+                if "amrfinderplus_scope" in feature.qualifiers:
+                    amr_track.genomic_features([feature], fc="dodgerblue")
+                if "defense_finder_type" in feature.qualifiers:
+                    antiphage_track.genomic_features([feature], fc="orchid")
+
+            elif feature.type in ["tRNA", "ncRNA"]:
+                rna_track.genomic_features([feature], fc="darkmagenta")
+            elif mobilome and feature.type in [
+                "mobility_island",
+                "cellular_recombinase",
+                "insertion_sequence",
+                "conjugative_element",
+                "nested_mobile_element",
+                "terminal_inverted_repeat_element",
+                "direct_repeat_element",
+                "viral_sequence",
+            ]:
+                mobilome_track.genomic_features([feature], fc="lightseagreen")
+            elif mobilome and feature.type.lower() == "phage":
+                mobilome_track.genomic_features([feature], fc="blue")
+
+    fig = circos.plotfig()
+    # Add legend
+    handles = [
+        Patch(color="hotpink", label="Forward CDS"),
+        Patch(color="steelblue", label="Reverse CDS"),
+        Patch(color="darkmagenta", label="RNA"),
+        Patch(color="tomato", label="BGCs (antiSMASH)"),
+        Patch(color="lightsalmon", label="BGCs (GECCO)"),
+        Patch(color="firebrick", label="BGCs (SanntiS)"),
+        Patch(color="forestgreen", label="Predicted PULs"),
+        Patch(color="dodgerblue", label="AMR genes"),
+        Patch(color="orchid", label="Anti-phage defense genes"),
+    ]
+    if mobilome:
+        handles = handles + [
+            Patch(color="blue", label="Mobilome (phage)"),
+            Patch(color="lightseagreen", label="Mobilome (other)"),
+        ]
+
+    main_legend = circos.ax.legend(
+        handles=handles, bbox_to_anchor=(0.5, 0.475), loc="center", fontsize=8
+    )
+    circos.ax.add_artist(main_legend)
+    fig.savefig(outfile, dpi=600)
```

Comment:
I would make the dpi configurable, as an optional arg. But not a blocker for this PR

Reply:
Good idea, added.